src/inc/teams.html

```diff
@@ -40,17 +40,19 @@ <h2 class="team-box__title">Team 2</h2>
     <!-- FOOTER -->
     <footer>
       <nav class="navbar">
-        <a href="../index.html"
-          ><img src="/design/icons/DashboardIcon.svg" alt="icon"
+        <a href="/src/index.html"
+          ><img class="active" src="/design/icons/DashboardIcon.svg" alt="icon"
```

Comment:
Since this is the teams.html, there doesn't need to be a class="active" on the dashboard img.
As you mentioned it is a copy past mistake to have another <a> tag inside.